hibernate-core/src/main/java/org/hibernate/engine/internal/StatefulPersistenceContext.java

```diff
@@ -1800,13 +1799,13 @@ private void managedSharedCacheEntries(
 							naturalIdCacheKey,
 							id,
 							null
-					);
+							);
 
 					if ( put && factory.getStatistics().isStatisticsEnabled() ) {
 						factory.getStatistics().naturalIdCachePut(
```

Comment:
@aepedraza I'm not sure i necessarily agree with a number of your formatting changes you've made here.  For example, if you compare the format of this method call with its older equivalent, you'll notice that the the closing `)` on the last line is left-aligned with the word `factory`.  This is generally how we generally format multiple lines with a closing brace on a separate line.  

Such cases like this have generated quite a number of diffs that I believe are unnecessary.

Reply:
Hi @Naros. I'm not sure about those formatting also. In fact, I disagree with some of them, specially the one you're pointing about closing brace in a new line.
I accepted them because I assume they were added due to the hibernate formatter for Eclipse, but I can review (and almost sure remove) those formatting and review the formatter afore mentioned, if the Hibernate team agree with this.
I'll be awaiting your comments. Best regards

Reply:
@aepedraza To be clear, I'm saying that the `)` goes on a new line, just its indention is incorrect.  

I'm not sure what you mean by _hibernate formatter for Eclipse_ .  We use check-style to verify that the format of the code is uniform to our desired layout.  

You can find the check-style rules in the `shared/config/checkstyle` directory in the root of the repository.  In master there are 2 files where one describes style rules that won't fail the build and another which will force the build to fail should any rules be broken.  I'd suggest you check both with your local changes.

Reply:
I expressed myself wrong, I meant I disagree with the indentation for braces and parenthesis in new lines.
By *hibernate formatter* I mean [this XML](https://raw.githubusercontent.com/hibernate/hibernate-ide-codestyles/master/eclipse/hibernate-java-formatting.xml) mentioned in the post [Contributing to Hibernate using Eclipse](https://developer.jboss.org/wiki/ContributingToHibernateUsingEclipse) (which is in the contributing guidelines).
I will review the check-style rules as you say and I'll be correcting the indentation (and the formatting in general)

Reply:
The IDE formatting guidelines are meant for new code.

Unfortunately IDEA and Eclipse - and other IDEs - don't have the capabilities to set exactly identical formatting settings, they all have their own quirks.

So the understanding is people shouldn't reformat existing code - unless something is severely broken of course.

You can see that otherwise I might open this same class in NetBeans in a week and change it back again, we'd never stop reformatting ;-)

Reply:
I see what you mean @Sanne, and agree with you.

So, to sumarize, I'll undo all the formatting hunks in my commits. And in the future I'll have an eye on automatic formatting.

Thanks for your time.